nss-day-cohort-29/reactive-nutshell-copperden
# Fixed News List display after delete

# Description
When you deleted an article and the News List was refreshed, ALL articles were being shown, not just the user's. Fixed the fetch so only the current user's articles are shown.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout meg-fix`
3. Make sure you have news articles in your database assigned to different userIds.
4. Go to "News" and delete an article.
5. Ensure that when the page reloads, the same articles are displayed (i.e. only the current user's, not anyone else's).

# Checklist:
